package/MDAnalysis/analysis/rms.py

```diff
@@ -484,6 +486,63 @@ def save(self, filename=None):
         return filename
 
 
+def rmsf(atomgroup, start=0, stop=-1, step=1, progout=None):
```

Comment:
The deed is done. It's now a class, though it's still rather bare-bones (no options for on-the-fly fitting). I agree with the reasoning that analyses that iterate through a trajectory are better constructed as classes, especially since we may be able to later stack them by pulling out their loop methods (once we have those...).